site/en/blog/chromium-chronicle-28/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "The Chromium Chronicle #28: Getting started with Chrome on iOS"
+description: >
+  Learn how to work with the code behind Chrome's iOS app.
+layout: 'layouts/blog-post.njk'
+date: 2022-01-25
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  The Chromium Chronicle
+tags:
+  - chromium-chronicle
+---
+
+**Episode 28:** by Mark Cogan in Paris, France (January, 2022)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+In 2022, Chrome on iOS will celebrate its 10th anniversary. 
+Chrome is one of the most widely used iOS apps&mdash;it's currently the number 2 Utility app in the US App Store. 
+
+Chrome on iOS provides all of the great Chrome features our users love, including Sync, Translate, Incognito, Password Manager, Autofill, and many more. 
+Chrome on iOS also integrates with native iOS features such as multi-window and device-wide password autofill. 
+
+However, there's one major thing that makes Chrome on iOS very different from all other Chrome platform implementations.
+
+Chrome on iOS doesn't use Blink. 
+
+Because of Apple's App Store rules, Chrome on iOS has to use the iOS platform API's for fetching and rendering web content. So Chrome on iOS uses WebKit, Nitro and CFNetwork where other Chrome platforms are using Blink, V8 and `//net`. Due to this, and the limited API available for iOS apps to interact with web page content, many Chrome features that use the `//content` APIs are implemented in Chrome on iOS using JavaScript injection. 
+
+Cross-platform features in `//components` are structured to share as much code as possible across all platforms while keeping `//content` dependencies siloed:
+
+```bash
+ls components/some_component
+
+content/        # code with dependencies on //content.
+core/           # cross-platform code with no //content dependencies.
+ios/            # iOS implementation using JavaScript injection.
+```
+
+Other important differences between Bling and other platforms include:
```

Comment:
Can we please also replace this instance of "Bling" with "Chrome in iOS" or an alternative word? Thanks!